tests/src/Kernel/Entity/FieldOverrideTest.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+/**
+ * @file
+ * Contains Drupal\Tests\og\Kernel\Entity\FieldOverrideTest.
+ */
+
+namespace Drupal\Tests\og\Kernel\Entity;
+
+use Drupal\Component\Utility\Unicode;
+use Drupal\field\Entity\FieldConfig;
+use Drupal\node\Entity\NodeType;
+use Drupal\KernelTests\KernelTestBase;
+use Drupal\og\Og;
+
+/**
+ * Testing field definition overrides.
+ *
+ * @group og
+ */
+class FieldOverrideTest extends KernelTestBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public static $modules = ['user', 'field', 'entity_reference', 'node', 'og', 'system'];
+
+  /**
+   * @var NodeType
+   */
+  protected $bundle1;
+
+  /**
+   * @var NodeType
+   */
+  protected $bundle2;
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function setUp() {
+    parent::setUp();
+
+    // Add membership and config schema.
+    $this->installConfig(['og']);
+    $this->installEntitySchema('og_membership');
+    $this->installEntitySchema('user');
+    $this->installEntitySchema('node');
+
+    // Create a two bundles.
+    $this->bundle1 = NodeType::create([
+      'type' => Unicode::strtolower($this->randomMachineName()),
+      'name' => $this->randomString(),
+    ]);
+    $this->bundle1->save();
+
+    $this->bundle2 = NodeType::create([
+      'type' => Unicode::strtolower($this->randomMachineName()),
+      'name' => $this->randomString(),
+    ]);
+    $this->bundle2->save();
+  }
+
+  /**
+   * Testing field definition overrides.
+   */
+  public function testSelectionHandlerResults() {
+    Og::CreateField(OG_AUDIENCE_FIELD, 'node', $this->bundle1->id(), ['instance' => ['label' => 'Other groups dummy']]);
```

Comment:
lets add another case with the simplest option (i.e. `Og::CreateField(OG_AUDIENCE_FIELD, 'node', $this->bundle1->id()`)